diff --git a/node_modules/react-native-select-dropdown/src/styles.js b/node_modules/react-native-select-dropdown/src/styles.js
index 4115f96..f1de8a9 100644
--- a/node_modules/react-native-select-dropdown/src/styles.js
+++ b/node_modules/react-native-select-dropdown/src/styles.js
@@ -25,7 +25,7 @@ const styles = StyleSheet.create({
     marginHorizontal: 8,
   },
   dropdownCustomizedButtonParent: {
-    flex: 1,
+    // flex: 1,
     overflow: 'hidden',
   },
   //////////////////////////////////////
